### ISSUE TITLE
Cache address and property

### ISSUE THREAD

**Issue by chrisjaure** [dhmncivichacks/isitrecycling#1]
Need to cache last used address and property to cut down on api calls.

**Comment by chrisjaure** [comment]
Property is no longer used so we just need to cache last used address, and maybe the specific endpoint to check.

**Comment by zo0o0ot** [comment]
What happened to everything that the property isn't used anymore?  Is it the API locator that eliminated the need for it?

**Comment by chrisjaure** [comment]
@zo0o0ot That and we added new endpoints to lookup by address since Appleton API is the only one that uses property so that's more of an implementation detail for that API.

**Comment by mikeputnam** [comment]
Ah yes - all the clients (web, Pebble, Android, future?) should be using http://civic-hack-api-locator.azurewebsites.net/#/contracts to derive their data.  That way when the next municipality is added, all the clients work with it!  Same for the next client, if they all adhere to the contract, they can all use the same data.

**Comment by zo0o0ot** [comment]
Yeah, I think the Pebble app is using that now.  @mrosack was nice enough to add that functionality here: https://github.com/zo0o0ot/AppletonPebble/pull/11 .  He's pretty awesome.
